### PR TITLE
[MAT-2270] Filling out code system URI Map

### PR DESCRIPTION
**Pull request by jkotanchik-SB**
**Expanding map**
- Covers code system listed at [https://cts.nlm.nih.gov/fhir/](https://cts.nlm.nih.gov/fhir/).
- Matches related maps in bonnie.

**Caching map**

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: lib/helpers/calculator_helpers.js [hunk 1]
```diff
@@ -495,4 +481,84 @@ module.exports = class CalculatorHelpers {
     };
     return elmFunction;
   }
+
+  static map;
+  static systemUriByName(systemName) {
+    if(this.map === undefined) {
```
Comment (jkotanchik-SB) [inline lib/helpers/calculator_helpers.js:487]: Is this an acceptable pattern for establishing a cache? 

Comparing the static property against `undefined` worked in unit tests, but I'm not sure if there any "gotchas" with it.

File: lib/helpers/calculator_helpers.js [hunk 2]
```diff
@@ -495,4 +481,84 @@ module.exports = class CalculatorHelpers {
...
+      /* eslint dot-notation:0 */
+      this.map = {};
+      this.map['medication-statement-status'] = 'http://hl7.org/fhir/CodeSystem/medication-statement-status';
+      this.map['medicationrequest-intent'] = 'http://hl7.org/fhir/CodeSystem/medicationrequest-intent';
+      this.map['medicationrequest-status'] = 'http://hl7.org/fhir/CodeSystem/medicationrequest-status';
+      this.map['discharge-disposition'] = 'http://terminology.hl7.org/CodeSystem/discharge-disposition';
```
Comment (serhii-ilin) [inline lib/helpers/calculator_helpers.js:493]: Probably we don't need that map in cqm-execution if I merge another PR which updated ValueSets with system URIs
